index.html

```diff
@@ -164,9 +180,9 @@ <h2>LET’S CONNECT</h2>
           <div class="img5"><img src="./image/Ellipse5.png" alt="img" ></div>
 
         
-               <div class="contact-form">
-                 
-                <h1 class="intro">Contact Me</h1>
+          <div class="contact-form">
+            <div class="res-contact">
+              <h1 class="intro">Contact Me</h1>
```

Comment:
- [ ] You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the info of the templates in Figma, you can change the ``` font-family``` to ``` Poppins```